editoast/assets/sprites/generate-atlas.sh

```diff
@@ -4,7 +4,9 @@
 # First add all your svg in a subfolder named to the signaling system (eg: `BAL`)
 # Then run this script. You will need docker.
 
```

Comment:
```suggestion
set -e
```

To make the script fail when one of the commands fails.